src/main/java/com/gargoylesoftware/htmlunit/README.md

```diff
@@ -1,5 +1,5 @@
 # Overview
-Jenkins used to maintain a forked version of HtmlUnit and use it via `JenkinsRule` (and `HudsonTestCase`). Moving
+Jenkins used to maintain a forked version of HtmlUnit and use it via `JenkinsRule` (and `JenkinsTestCase`). Moving
```

Comment:
Should not rename the HudsonTestCase class, so should not change this documentation in this way
```suggestion
Jenkins used to maintain a forked version of HtmlUnit and use it via `JenkinsRule` (and `HudsonTestCase`). Moving
```